api/camunda/camunda-openapi.yaml

```diff
@@ -1,5 +1,9 @@
 # Disclaimer: This is a modified version of the Camunda REST API specification, optimized for the documentation.
 
+# Disclaimer: This is a modified version of the Camunda REST API specification, optimized for the documentation.
+
+# Disclaimer: This is a modified version of the Camunda REST API specification, optimized for the documentation.
+
```

Comment:
Hmm, interesting that we have this duplicated in here now 😅 
Not a big deal, of course, it's just a comment in the spec file in the docs, not the source.

Reply:
oh, fun! I know why that happened. I re-generated the docs in this branch, and it added that comment both times. 

I'll fix this for the future!